extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/runtime/RedisClientImpl.java

```diff
@@ -1017,6 +1021,17 @@ public Response zscore(String arg0, String arg1) {
         return await(redisAPI.zscore(arg0, arg1));
     }
 
+    @Override
+    public Response zunion(List<String> args) {
+        final io.vertx.mutiny.redis.client.Command ZUNION = Command.create("zunion", -3, 0, 0, 0, false, true, true, false);
+        final io.vertx.mutiny.redis.client.Request requestWithArgs = args.stream().reduce(
+                io.vertx.mutiny.redis.client.Request.cmd(ZUNION),
+                (request, s) -> request.arg(s),
+                (request, request2) -> request);
+
```

Comment:
Yes this is the valid approach. 

The only trick part to explain can be the creation of new command. Is there some Vertx links we can add in the documentation?

Reply:
@machi1990 For this purpose I would describe the procedure based on running an node app (available in `vertx-redis-client`) to generate the helper methods, as described by @pmlopes here: https://github.com/vert-x3/vertx-redis-client/issues/299#issuecomment-873386056.

So I think this is covered :)